GitDepend.UnitTests/Visitors/RemoveDependencyVisitorTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitDepend.Configuration;
+using GitDepend.Visitors;
+using NUnit.Framework;
+
+namespace GitDepend.UnitTests.Visitors
+{
+    [TestFixture]
+    public class RemoveDependencyVisitorTests : TestFixtureBase
+    {
+        private RemoveDependencyVisitor _visitor;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            _visitor = new RemoveDependencyVisitor("lib1");
+        }
+
+        [Test]
+        public void RemoveDependencyShouldFindDependency()
+        {
+            var result = _visitor.VisitDependency("lib1", new Dependency()
```

Comment:
this looks like it's not correct. The first argument is the directory of the project.

This should probably be
```
_visitor.VisitDependency(Lib2Directory, Lib1Dependency)
```